.github/workflows/ci-beta.yml

```diff
@@ -0,0 +1,76 @@
+name: CI with Gradle Beta
```

Comment:
probably can be a better name

Reply:
Agree, that said a better name is evading me. Options could just be "CI Build" or "Verification Pipeline"